stdlib/ballerina-http/src/main/ballerina/http/websocket_connector.bal

```diff
@@ -56,17 +56,20 @@ public type WebSocketConnector object {
     }
     public native function pong(blob data) returns error|();
 
-    @Description {value: ""}
-    @Param {value: "statusCode: "}
-    @Param {value: "reason: "}
     documentation {
         Close the connection.
 
         P{{statusCode}} Status code for closing the connection
         P{{reason}} Reason for closing the connection
+        P{{timeoutInSecs}} timeout which waits for the close frame from remote endpoint to close the connection.
```

Comment:
Rather timeoutInSec is more appropriate and commonly used I suppose. Shall we remove the tailing s?

Reply:
Any particular reason for using seconds? We use millis in other places for timeout.